tests/epyccel/test_epyccel_classes.py

```diff
@@ -0,0 +1,11 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring, missing-class-docstring
+from pyccel.epyccel import epyccel
+
+def test_class_import(language):
+    class A:
+        def __init__(self : 'A'):
+            pass
+
+    epyc_A = epyccel(A)
+
+    assert isinstance(epyc_A, type)
```

Comment:
Shouldn't `epyc_A` be of type `A` just like in Python? If not, code that checks `isinstance(obj, A)` will fail to work after pyccelization.

Reply:
No. We are returning the Pyccelised class A not an instance of that class so the result must be a type. If we want the class methods to call Pyccelised functions I don't think the resulting class can be equivalent to A

Reply:
Ah sorry, this is not an instance of A but A itself!